app/config.py

```diff
@@ -22,6 +29,38 @@
 
 TIME_TO_CANCEL_ACTIONS = 60
 
+DEFAULT_DURATION = timedelta(hours=1)  # длительность ограничения по умолчанию
```

Comment:
I don't like the idea of storing `timedelta` objects in the config. Better in numbers, 
for example:

`DEFAULT_RESTRICT_DURATION_HOURS = 1`

Reply:
I'll think about it. Maybe it will be not actually after refactoring config.py
I think for hardcoded config `timedelta` is ok, and when config will be object (`dict` or `dataclass`) in his creation will be a lot processing code, and in that place will be convert from numbers to `timedelta`